plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -2747,6 +2747,7 @@ function woocommerce_order_again_button( $order ) {
 			'order/order-again.php',
 			array(
 				'order'           => $order,
+				'wp_button_class' => wc_wp_theme_get_element_class_name( 'button' ) ? ' ' . wc_wp_theme_get_element_class_name( 'button' ) : '',
```

Comment:
```suggestion
				'wp_button_class' => wc_wp_theme_get_element_class_name( 'button' ),
```

Optional, non-blocking suggestion, but I would argue for having this template parameter only contain the classname, and worry about the whitespace in the template itself. And also, probably not worry about extra whitespace at the end of the `class` attribute in the case that `wp_button_class` is empty.

Reply:
I was inspired by other instances of the same fix throughout the codebase (like [here](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/wc-template-functions.php#L3247)) 

To be honest, I also believe that the templates should be responsible for managing the classes. Ideally, I'd expect to _echo_ them using arrays and the `implode` function. We could move forward with this as _is_, progress with other changes in regards to the _block-ification_ of the templates, and then return to create a more organized solution. 

I'd be happy to change it, though. Thoughts?

Reply:
> Ideally, I'd expect to echo them using arrays and the implode function. We could move forward with this as is, progress with other changes in regards to the block-ification of the templates, and then return to create a more organized solution.

👍 Yeah, doing this separately sounds like the right approach. No need to make this PR more complicated.